data/human/human missions.txt

```diff
@@ -7455,3 +7455,1848 @@ mission "Adelita 5"
+			choice
+				`	(Head back to the spaceport.)`
+					goto options
+			
+			label hookmeup
+			`	The three of them look at each other and then give you a carnivorous grin.`
+			`	"Sure thing," he smirks. "Easy as rain. It'll cost ya 'bout 20,000 cred." They look at you expectantly.`
+			choice
+				`	(Give them the credits.)`
+					goto buyid
+				`	"Wait! Show me how you're going to do it first."`
+				`	(Turn them down.)`
+					goto refuseid
+
+			`	He sighs. "Fine. I know a guy in the city, he snags us fake IDs all the time, gets us into bars, clubs, all sorts. All we need is how ya want this false guy to look, we'll take it to him and get the ID, and you'll be sweet.`
+			`	"So, ya gonna take it?"`
+			choice
+				`	(Give them the credits.)`
+					goto buyid
+				`	(Turn them down.)`
+
+			label refuseid
+			`	Collectively they frown. "Your loss," the middle guy says, before leaving you alone to return to the spaceport.`
+				goto options
+
+			label buyid
+			action
+				payment -20000
+			`	"Alrighty then, it's a done deal," he says, stashing the credit chip away. He then pulls out a datapad and asks you a few questions about what to put on the ID. You try to answer as best you can in regards to Timothy (and change a few of the more obviously incriminating details). They say to meet you here in a couple hours once they've got the ID sorted in the city.`
+			choice
+				`	(Wait.)`
+
+			`	Two and a half hours pass as you wait for the teens in the garden, but there's no sign of them yet. You wonder if you've just given 20,000 credits away for nothing.`
+			choice
+				`	(Leave.)`
+					goto leaveid
+				`	(Wait.)`
+
+			`	Another thirty minutes pass, but just as you're about to leave the three teens show themselves again, ID in hand.`
+			`	"We thought you're gonna bail out on us for bein' late. Thankfully you got the sense not to. And now ya get the goods in exchange." The middle kid hands you the ID. "Good dealin' with ya," he says as they walk off.`
+			`	As you go back to the spaceport, you take a look at the ID for the first time.`
+			`	Whoever made this clearly wasn't a professional. It looks like they've somehow botched the bar code, given the smear on the print, and the duplication of the emblem of the planetary authority of New Britain on the back is rather dodgy. But unless you can find a better alternative, it'll have to do.`
+				goto options
+
+			label leaveid
+			`	You sigh, having learnt a lesson that teenagers in parking lots probably aren't the most trustworthy of people. Back to the spaceport.`
+				goto options
+
+			label end
+			action
+				clear "med1"
+				clear "med2"
+				clear "suit1"
+				clear "suit2"
+				clear "id1"
+			`	After a long day of walking about the New British spaceport and wading through the crowds of hustlers and travellers, you return to the relatively calmer terminal area at the top of the pyramid.`
+				decline
+
```

Comment:
These missions probably shouldn't continue indefinitely once you've finished or failed to save Tim since you only need to complete one or two of them. I _think_ the below should do it
```suggestion
	to fail
		or
			has "Timothy Radrickson 5a: Timshank Redemption: done"
			has "Timothy Radrickson 5a: Timshank Redemption: failed"

```